README.md

```diff
@@ -1,14 +1,50 @@
-This is a [Giter8][g8] template for Akka in Scala.
+# Akka Quickstart with Scala
 
+## Quickstart guide
+
+This example is described in the [Akka Quickstart with Scala guide](http://developer.lightbend.com/guides/akka-quickstart-scala/)
+
+The Hello World example illustrates Akka basics. Within 30 minutes, you should be able to download and run the example and use this guide to understand how the example is constructed. This will get your feet wet, and hopefully inspire you to dive deeper into the wonderful sea of Akka!
+
+## Giter8 template
+
+It can also be used as a [Giter8][g8] template for Akka with Scala.
+
+Prerequisites:
+- JDK 8
+- [sbt][sbt] 0.13.13 or higher ([download here][sbt_download])
+
+Open a console and run the following command to apply this template:
+ ```
+sbt -Dsbt.version=0.13.15 new https://github.com/akka/akka-scala-seed.g8
```

Comment:
It will result in errors and users will go elsewhere (and I have heard that those errors are not that easy to understand). When newer sbt launchers are more spread that problem will go away.

I see these as copy paste text so it doesn't really matter how many chars.

@eed3si9n What is your recommendation for including `-Dsbt.version=0.13.15` in readme instructions or not ?